external-resolver/src/main/scala/org/genivi/sota/resolver/db/DeviceRepository.scala

```diff
@@ -109,7 +109,11 @@ object DeviceRepository {
                     (implicit ec: ExecutionContext): DBIO[Unit] =
     for {
       pkg <- PackageRepository.exists(namespace, pkgId)
-      _ <- installedPackages.insertOrUpdate((device, pkg.uuid))
+      _ <- installedPackages.insertOrUpdateWithKey((device, pkg.uuid),
+             all => all
```

Comment:
So I don't know what is supposed to happen here. Insert and if it exists already, do nothing?
that could be done with `asTry` and ignore on failure if the error is a constraint error.